packages/mongodb-chatbot-server/src/routes/content/searchContent.ts

```diff
@@ -58,25 +64,39 @@ interface SearchContentResponseBody {
 export function makeSearchContentRoute({
   findContent,
   searchResultsStore,
+  addCustomData,
 }: MakeSearchContentRouteParams) {
+  const tracedFindContent = wrapTraced(findContent, { name: "searchContent" });
   return async (
     req: ExpressRequest<SearchContentRequest["params"]>,
     res: ExpressResponse<SearchContentResponseBody, SearchContentRouterLocals>
   ) => {
     try {
+      // --- INPUT VALIDATION ---
+      const { error } = SearchContentRequestBody.safeParse(req.body);
```

Comment:
```suggestion
      const { error, data } = SearchContentRequestBody.safeParse(req.body);
```

i think better to use the parsed data isntead of raw req.body. this way if we add more sophisticated data validation/tranformation in the zod type, it'll already be captured.